homeassistant/components/acmeda/sensor.py

```diff
@@ -41,15 +42,15 @@ def async_add_acmeda_sensors():
 class AcmedaBattery(AcmedaBase, SensorEntity):
     """Representation of a Acmeda cover device."""
 
-    device_class = SensorDeviceClass.BATTERY
+    _attr_device_class = SensorDeviceClass.BATTERY
     _attr_native_unit_of_measurement = PERCENTAGE
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of roller."""
         return f"{super().name} Battery"
 
     @property
-    def native_value(self):
+    def native_value(self) -> float | None:
```

Comment:
I'm assuming this is a float, but I was unable to check.
Maybe it is an int?
And can it be `None`?
```suggestion
    def native_value(self) -> int | None:
```

Reply:
I guess we could use `StateType` or be extra safe with `float | int | None`. In the end, this is just typing and not critical at runtime.

Reply:
@atmurray could you confirm the possible values?

Reply:
Since the code owner is not replying, I have set it `float | int | None`